pybaseball/plotting.py

```diff
@@ -87,40 +102,43 @@ def spraychart(data, team_stadium, title='', tooltips=[], size=100,
         legend_title = 'Player'
     else:
         color_label = colorby
+
     # scatter plot of hits
-    scatter = alt.Chart(sub_data, title=_title).mark_circle(size=size).encode(
-        x=alt.X('hc_x:Q', axis=None, scale=alt.Scale(zero=True)),
-        y=alt.Y('y:Q', axis=None, scale=alt.Scale(zero=True)),
-        color=alt.Color(
-            color_label, legend=alt.Legend(title=legend_title)
-        )
-    ).transform_calculate(
-        y='datum.hc_y * -1'
-    )
+    scatters = []
+    for color in sub_data[color_label].unique():
+        color_sub_data = sub_data[sub_data[color_label] == color]
+        scatters.append(base.scatter(
+            color_sub_data["hc_x"], color_sub_data['hc_y'].mul(-1), size, label=color, alpha=0.5
+        ))
+
     if tooltips:
-        scatter = scatter.encode(tooltip=tooltips)
+        warnings.warn(
+            "Tooltips are disabled in the pyplot version of spraychart and will be removed in the future",
+            category=DeprecationWarning
+        )
+
+    plt.legend(handles=scatters, title=legend_title, bbox_to_anchor=(1.05, 1), loc='upper left')
 
-    plot = alt.layer(base, scatter).resolve_scale(color='independent')
-    plot.width = width
-    plot.height = height
-    del sub_data
+    plt.draw()
 
-    return plot
+    plt.show()
 
+    # return plot
+    return base
 
-def plot_bb_profile(df: pd.DataFrame, parameter: Optional[str] = "launch_angle"):
+
+def plot_bb_profile(df: pd.DataFrame, parameter: Optional[str] = "launch_angle") -> None:
     """Plots a given StatCast parameter split by bb_type
 
     Args:
-        df (pd.DataFrame): StatCast pd.DataFrame (retrieved through statcast, statcast_batter, etc)
+        df (pd.DataFrame):                   StatCast pd.DataFrame (retrieved through statcast, statcast_batter, etc)
```

Comment:
nitpick: wacky indent

Reply:
I tried making all the docstrings in this file like the google style + types that @jldbc and I had talked about in #125 . Let me know what you think.